nextjs-todolist-app/README.md

```diff
@@ -0,0 +1,237 @@
+
+Choreo is an Internal Developer Platform (IDevP) that allows you to build, deploy, monitor, and manage your cloud-native applications effortlessly.
+
+In this quick start guide, you will explore how to expose a service endpoint as a REST API via Choreo and securely consume the API from a Next.js web application.
+
+You will build a simple todo list web application with a sign-in page and functionality to interact with a secure backend REST API. You will use Asgardeo, WSO2's Identity as a Service (IDaaS) solution, to secure user authentication to the web application. The application will allow users to sign in and view the todo list. On signing in, a user can view profile details and view todo lists and view description of the task separately. The application will also allow users to sign out of the application.
+
+This guide walks you through the following steps:
+
+- Develop, deploy the service component.
+- Securely consume the golang service via your web application.
+- Deploy your web application and use Asgardeo as the IDaaS provider to secure user authentication to the web application.
+
+## Prerequisites
+
+Before you try out this guide, complete the following:
+
+1. Create a GitHub repository to save the service implementation. For this guide, you can fork [https://github.com/wso2/choreo-samples.git](https://github.com/wso2/choreo-samples.git).
+2. If you are signing in to the Choreo Console for the first time, create an organization as follows:
+
+    1. Go to [https://console.choreo.dev/](https://console.choreo.dev/), and sign in using your Google, GitHub, or Microsoft account.
+    2. Enter a unique organization name. For example, `Stark Industries`.
+    3. Read and accept the privacy policy and terms of use.
+    4. Click **Create**.
+
+    This creates the organization and opens the home page of the default project created for you.
+
+    !!! info "Enable Asgardeo as the key manager"
+
+         If you created your organization in Choreo before the 21st of February 2023, and you have not already enabled Asgardeo as the key manager, follow these steps to enable Asgardeo as the default key manager:
+
+         1. In the Choreo Console, go to the top navigation menu and click **Organization**. This takes you to the organization's home page.
+         2. In the left navigation menu, click **Settings**.
+         3. Click the **API Management** tab and then click **Enable Asgardeo Key Manager**.
+         4. In the confirmation dialog that opens, click **Yes**.
+
+## Step 1: Create the service component
+
+In this step, you are playing the role of an API developer. You will create a service and deploy it as a REST API for your web application to consume.
+
+### Step 1.1: Create the service
+
+Follow the steps below to create the service:
+
+1. Go to [https://console.choreo.dev/](https://console.choreo.dev/cloud-native-app-developer) and sign in. This opens the project home page.
```

Comment:
Shall we change the URL since the `cloud-native-app-developer part` is not in use